src/main/java/Problem1/MinStack.java

```diff
@@ -1,31 +1,49 @@
 package Problem1;
 
-public class MinStack extends ArrayStack<Integer> {
+public class MinStack<T> extends ArrayStack<Integer> {
```

Comment:
do not change interface like this. if you believe you have a reason, please check with instructor first.